asm/macros/event.inc

```diff
@@ -1939,3 +1939,12 @@
 	setvar VAR_0x8006, \item
 	special CreateEventLegalEnemyMon
 	.endm
+
+	@ Attempts to trigger a special evolution method.
+	@ 'evoMethod' is the evolution you want to trigger.
+	@ There may be other conditions required which are coded for in GetEvolutionTargetSpecies.
+	@ EX: tryspecialevo EVO_WATER_SCROLL will trigger an evolution for Kubfu.
+	.macro tryspecialevo evoMethod:req
+	setvar VAR_0x8000, \evoMethod
+	special TrySpecialOverworldEvo
+	.endm
```

Comment:
You forgot to define the `TrySpecialOverworldEvo` special.

It works well, but I think we should add options for letting the evolution being cancelable by pressing B. I wasn't able to test if Kubfu's could be canceled, but you can with Galarian Yamask, so I feel it'd be best to leave that descition to the user, if they ever want to force it.

Another option that I feel should be good for users would be letting them decide if the triggered evolution will affect just 1 Pokémon or everything in the party, as tested in SwSh, every Galarian Yamask in the party will evolve one after the other by a single trigger.

Also, don't forget to end the file with a line break.